apps/cognito-functions/src/templates/post-confirmation-confirm-sign-up-message.ts

```diff
@@ -0,0 +1,121 @@
+import { COMPANY_LOGO } from './company-logo';
+import mjml2html from 'mjml';
+import { minify } from 'html-minifier';
+
+const TRANSLATIONS: { readonly [k: string]: string | readonly string[] } = {
```

Comment:
What are the pros to define the type `{ readonly [k: string]: string | readonly string[] }` instead use the inferred one?